src/helperFunctions/toast.ts

```diff
@@ -0,0 +1,13 @@
+import { toast } from 'react-toastify';
+
+type ToastType = 'info' | 'success' | 'warning' | 'error' ;
+const toastContainer = (type: ToastType, message: string) => toast[type](message, {
+  position: toast.POSITION.TOP_RIGHT,
+  autoClose: 3000,
+  hideProgressBar: false,
```

Comment:
Please check what options are there by default with **react-toastify** and probably we can remove them. For example, if **hideProgressBar** is **false** by default we can avoid writing it explicitly